openmetadata-ui/src/main/resources/ui/src/components/Explore/ExploreSkeleton.component.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { Col, Row, Skeleton } from 'antd';
+import { uniqueId } from 'lodash';
+import { SkeletonInterface } from 'Models';
+import React from 'react';
+
+const ExploreSkeleton = ({ children, loading }: SkeletonInterface) => {
+  const mockData = Array.from({ length: 6 }, (_, index) => {
+    return {
+      name: index,
+      innerData: Array.from({ length: 3 }, (_, index) => index + 1),
+    };
+  });
```

Comment:
useMemo or move to utils file and make it generic so in future we can use this to create a grid for skeleton

Reply:
@ShaileshParmar11 , Here we are not doing any heavy computation so I think it's fine without `useMemo`.
Regarding the utils, we can do that if there is a need for it.